lib/bulk_processor/back_end/gcp.rb

```diff
@@ -7,6 +7,9 @@ def initialize(processor_class:, payload:, key:)
         @processor_class = processor_class.name
         @payload = PayloadSerializer.serialize(payload)
         @key = key
+
+        GcpManager.add_task('start-bulk-processor', 'rake bulk_processor_gcp_pods:start')
+        GcpManager.add_task('split-bulk-processor', 'rake bulk_processor_gcp_pods:split')
```

Comment:
this seems like the wrong place. We want it to be added only once, not every time the instance is created.

Reply:
@jeffnoehren  ^^^

Reply:
@abezzub yea good point i will move it